README.md

```diff
@@ -425,6 +425,16 @@ Furthermore, CTL exposes an `overlay` from its flake. You can use this in the Ni
 
 We have recenly set up a small scaffolding repository for projects wishing to adopt CTL: https://github.com/mlabs-haskell/ctl-scaffold. More documentation and resources will be added soon to the repo
 
+### Changing network configurations.
+
+CTL supports using networks other than testnet to provide development environment.
+
+First you need to specify alternative way of getting the network configuration.
+
+[cardano-configurations](https://github.com/input-output-hk/cardano-configurations) repo provides configs for `mainnet`, `staging` and a few other networks. If you want one of these, simply replace `network` parameter in `buildCtlRuntime`.
+
+Providing custom repo with similar structure is also possible: edit `flake.nix` and change `inputs.cardano-configurations` providing the URL of your fork. You can use `path:/path/to/repo` as URL as well. After that, edit `network` parameter (it should point to a subdirectory of `./network/` in your fork).
```

Comment:
I think it would be better to illustrate something like 
```
inputs.cardano-transaction-lib.inputs.cardano-configurations.follows = "...";
```

since users can't edit our flake unless they fork the repo or point it towards a local copy, which would be very inconvenient

Also, I think it might be better to use `git+file://...` as an example for a local directory as IIRC `path:...` requires the input to be a flake 

See my other comments below about adding a configurations directory as an argument for building the CTL runtime